int32[] ids
string[] colors
float64[] alphas
---
